src/layouts/BaseLayout.astro

```diff
@@ -84,6 +84,12 @@ const headerTopic = topic
       const htmlElement = document.documentElement;
       htmlElement.className = `${storedSettings.join(" ")} ${htmlElement.className}`;
     </script>
+    <link
+      rel="stylesheet"
+      href="https://foundation-donate-banner.netlify.app/static/css/main.css"
```

Comment:
@Stefterv @SableRaf The CSS here is overwriting the CSS variable used on the website to render monospace font `--font-serif`. Can the variable be scoped or if it is not used at all be removed from the CSS served?

Reply:
Just saw that this was already mentioned in #649

Reply:
Hey Kenneth! Thank you for the heads up, we've pushed a fix to the banner repo to fix this